string joint_name
string[] child_joints
float32[] uv  # image coordinates [u, v]
float32[] xyz  # real-world coordinates [x, y, z]
